easylist/easylist_general_hide.txt

```diff
@@ -21271,6 +21271,7 @@
 ##.l-container > #fishtank
 ! Google
 ###atvcap + #tvcap > .mnr-c > .commercial-unit-mobile-top
+###atvcap > div:has(h3[role="heading"] > span:has-text(/^Ads/))
```

Comment:
- `:has-text(foo)` and `:has(foo)` is no supported by ABP/AdBlock

- omitted domains is no supported by ABP/AdBlock for procedural filters (like `:-abp-has(foo)`)

   - also no supported simplify procedural filters to only TLD like `com##...`